samples/core/src/test/resources/config/application.properties

```diff
@@ -13,9 +13,14 @@ database.migration.auto=false
 database.migration.testdata=false
 database.migration.clean=false
 
+
+# Default values for job and step related properties
+chunk.size=2
```

Comment:
is this chunk size configuration just for the staffmember import job?
If so, please choose a more speaking property name, e.g., `batch.staffimport.chunk.size`